connect_ext_test.go

```diff
@@ -1988,13 +1988,14 @@ func TestBidiOverHTTP1(t *testing.T) {
 		server.URL(),
 	)
 	stream := client.CumSum(context.Background())
+	// Stream creates an async request, can error on Send or Receive.
 	if err := stream.Send(&pingv1.CumSumRequest{Number: 2}); err != nil {
 		assert.ErrorIs(t, err, io.EOF)
 	}
 	_, err := stream.Receive()
 	assert.NotNil(t, err)
 	assert.Equal(t, connect.CodeOf(err), connect.CodeUnknown)
-	assert.Equal(t, err.Error(), "unknown: HTTP status 505 HTTP Version Not Supported")
+	assert.True(t, strings.HasSuffix(err.Error(), "HTTP status 505 HTTP Version Not Supported"))
```

Comment:
There's no `ErrorContains` so kept the old behaviour.

Reply:
Weird. There is a `require.ErrorContains`. I wonder why they don't have symmetry in the `assert` package :/

Edit: er, wait. This module is using its own assert package instead of "stretchr/testify". We could always add such a function 🤷